src/5_group_families.js

```diff
@@ -1,3 +1,7 @@
-module.exports = function groupFamilies() {
-  // TODO Implement me.
+module.exports = function groupFamilies(ObjArr) {
+  const families = {}
+  for(const innerObj of ObjArr) {
+      families[innerObj.surname] ? families[innerObj.surname].push(innerObj.name) : families[innerObj.surname] = [innerObj.name]
```

Comment:
trenary z side effectem: https://stackoverflow.com/questions/55349332/using-ternary-operator-just-for-the-side-effect-good-or-bad